### DISCUSSION TITLE
make fails @ `usb.c.obj`

### DISCUSSION THREAD

**Discussion by iot49** [micropython/micropython#13522] [ESP32]
I am suddenly experiencing problems compiling micropython for the esp32**s3**:

```
[1279/1309] Building C object esp-idf/main_esp32s3/CMakeFiles/__idf_main_esp32s3.dir/__/usb.c.obj
FAILED: esp-idf/main_esp32s3/CMakeFiles/__idf_main_esp32s3.dir/__/usb.c.obj 
```

Has anyone else experienced issues? The error is specific to the s3 target; perhaps not surprising since that's the cpu with integrated USB.

Command used to compile:

```bash
docker run --rm -v .:/project -w /project espressif/idf:v5.0.4 \
    bash -c "cd micropython/ports/esp32; make clean; make submodules; make V=1 IDF_TARGET=esp32s3"
```

**Comment by jimmo** [comment]
@iot49 `IDF_TARGET` is not a supported argument on the esp32 Makefile?

If you're building for s3 then you need to set a board, e.g. `BOARD=ESP32_GENERIC_S3`

**Comment by iot49** [comment]
Many thanks! How stupid of me.
